frontend/package.json

```diff
@@ -46,6 +46,7 @@
     "babel-core": "^6.26.0",
     "babel-minify-webpack-plugin": "^0.3.1",
     "babel-preset-env": "^1.6.1",
+    "copy-webpack-plugin": "^4.5.1",
```

Comment:
[`html-webpack-plugin`](https://github.com/jantimon/html-webpack-plugin) has the option to add favicon. We dont need this plugin

Reply:
But if we have those all icons maybe we need it